dawn/src/dawn/Compiler/DawnCompiler.h

```diff
@@ -20,12 +20,31 @@
 #include "dawn/Optimizer/OptimizerContext.h"
 #include "dawn/Support/DiagnosticsEngine.h"
 #include "dawn/Support/NonCopyable.h"
+#include <list>
 #include <memory>
 
 namespace dawn {
 
 struct SIR;
 
+/// @brief Enumeration of all pass groups
+enum class PassGroup {
```

Comment:
Like the enumeration.

Reply:
That came from your enumeration in `CompilerUtil`. Good work! 👍